examples/guided.py

```diff
@@ -342,18 +455,34 @@ def perform_installation(mountpoint):
 				installation.enable_service('systemd-resolved')
 
 			if archinstall.arguments.get('audio', None) != None:
-				installation.log(f"This audio server will be used: {archinstall.arguments.get('audio', None)}", level=logging.INFO)
+				installation.log(
+					f"This audio server will be used: {archinstall.arguments.get('audio', None)}",
+					level=logging.INFO,
+				)
 				if archinstall.arguments.get('audio', None) == 'pipewire':
 					print('Installing pipewire ...')
 
-					installation.add_additional_packages(["pipewire", "pipewire-alsa", "pipewire-jack", "pipewire-media-session", "pipewire-pulse", "gst-plugin-pipewire", "libpulse"])
+					installation.add_additional_packages(
+						[
+							"pipewire",
+							"pipewire-alsa",
+							"pipewire-jack",
+							"pipewire-media-session",
+							"pipewire-pulse",
+							"gst-plugin-pipewire",
+							"libpulse",
+						]
+					)
 				elif archinstall.arguments.get('audio', None) == 'pulseaudio':
 					print('Installing pulseaudio ...')
 					installation.add_additional_packages("pulseaudio")
 			else:
 				installation.log("No audio server will be installed.", level=logging.INFO)
-			
-			if archinstall.arguments.get('packages', None) and archinstall.arguments.get('packages', None)[0] != '':
+
+			if (
+				archinstall.arguments.get('packages', None)
+				and archinstall.arguments.get('packages', None)[0] != ''
+			):
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability.